src/Microsoft.DocAsCode.Build.RestApi/Swagger/Internals/SwaggerObjectConverter.cs

```diff
@@ -37,10 +37,13 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
                             JToken existing;
                             if (jObject.TryGetValue(k.Key, out existing))
```

Comment:
Resolved

---
In reply to: [63802675](https://github.com/dotnet/docfx/pull/318#discussion_r63802675) [](ancestors = 63802675)